### DISCUSSION TITLE
Green-bot v9

### DISCUSSION THREAD

**Discussion by pauldb09** [GreenBotDevelopment/Green-bot#103] [General]
**Lavalink Version**

Hello everyone! As promished here is the new version with lavalink!

You will need to start a lavalink server and complete the informations in the config.js files. 

[Guide: How to start and create a lavalink server](https://dsharpplus.github.io/articles/audio/lavalink/setup.html)

Once your server is ready go in the `config.js` file and add a node.

```js
lavalink: [
        { name: "node", url: "IP:PORT", auth: "LAVALINK_PASSWORD" },
 ],
```

**Full Changelog**: https://github.com/pauldb09/Green-bot/compare/v8...v9

<hr /><em>This discussion was created from the release <a href='https://github.com/pauldb09/Green-bot/releases/tag/v9'>Green-bot v9</a>.</em>

**Comment by Saud-97** [comment]
Do you support "Lavalink with ssl" and "Lavalink without ssl".

I'm interested in using public ones like https://lavalink.darrennathanael.com/NoSSL/lavalink-without-ssl/

Would be preferable to also use same format as they use in other places like:
```
Host : lavalink.islantay.tk
Port : 8880
Password : "waifufufufu"
secure: false
```

**Reply by pauldb09** [reply]
> Do you support "Lavalink with ssl" and "Lavalink without ssl".
> 
> I'm interested in using public ones like https://lavalink.darrennathanael.com/NoSSL/lavalink-without-ssl/
> 
> Would be preferable to also use same format as they use in other places like:
> 
> ```
> Host : lavalink.islantay.tk
> Port : 8880
> Password : "waifufufufu"
> secure: false
> ```

Yeah of course it will work, you can add the secure option on the lavalink array, added in [this commit](https://github.com/pauldb09/Green-bot/commit/e1f9a183e32a1c409d06c4bd77936cf09faab04d)
